beacon_node/beacon_chain/src/beacon_chain.rs

```diff
@@ -190,7 +186,7 @@ impl<T: BeaconChainTypes> BeaconChain<T> {
             genesis_block_root: self.genesis_block_root,
             ssz_head_tracker: self.head_tracker.to_ssz_container(),
             fork_choice: self.fork_choice.as_ssz_container(),
-            block_root_tree: self.block_root_tree.as_ssz_container(),
+            block_root_tree: vec![],
```

Comment:
Should be deleted entirely (perhaps you had this in as a backwards compatibility shim)

Reply:
Yep, was a shim but it's no longer needed. Has been removed :)